File: tests/fixtures/v2-http-sources/positive/request-sources.js
function review(req, models) {
  const body = req.body.email;
  const query = req.query.q;
  const path = req.params.id;
  const header = req.headers["x-trace"];
  const cookie = req.cookies.session;
  models.sequelize.query(query);
}

File: tests/fixtures/v2-http-sources/positive/RequestSources.cs
class RequestSources
{
    void Review(dynamic Request)
    {
        var query = Request.Query["q"];
        var body = Request.Form["email"];
        var route = Request.RouteValues["id"];
        var header = Request.Headers["X-Trace"];
        var cookie = Request.Cookies["session"];
    }
}
